tools/minikube/scripts/start_pods.sh

```diff
@@ -73,6 +92,12 @@ kubectl create configmap \
     --from-literal=PINAKES_CONTROLLER_TOKEN="$PINAKES_CONTROLLER_TOKEN" \
     --from-literal=PINAKES_CONTROLLER_VERIFY_SSL="$PINAKES_CONTROLLER_VERIFY_SSL"
 
+kubectl create configmap \
```

Comment:
@hsong-rh Is this needed can we just use  the env vars or the default settings value